src/org/traccar/protocol/TelicProtocolDecoder.java

```diff
@@ -105,4 +110,57 @@ protected Object decode(
         return position;
     }
 
+    private String decodeAlarm(String event) {
+
+        int eventId = 0;
+
+        try {
+            eventId = Integer.parseInt(event);
+        } catch (Exception e) {
+            Log.error("TelicProtocolDecoder: Error parsing event (" + eventId + "): " + e.toString());
+        }
+
+        switch (eventId) {
+            case 1:
+                return Position.ALARM_POWER_ON; // Power on
```

Comment:
Don't add obvious comments like this. Basically all known alarms don't require any extra comment.